include/openenclave/attestation/sgx/attester.h

```diff
@@ -37,26 +37,6 @@ typedef void* oe_sgx_plugin_opt_params;
  */
 oe_attester_t* oe_sgx_plugin_attester(void);
```

Comment:
This function is no longer defined.

Reply:
Yes this file will be removed, and a new function oe_result_t oe_attester_initialize(void) will be defined in include/openenclave/attestation/attester.h